docs/assets/templates/workflows/workflows.rst

```diff
@@ -10,11 +10,11 @@ Simple generic template for a workflow
 .. include:: /assets/templates/workflows/template-small.dockstore.yml
   :code:
 
-Always use :ref:`absolute paths <dict absolute path>` to specify the :ref:`primary descriptor <dict primary descriptor file>` and :ref:`test parameter <dict parameter file>` files.
+Always use :ref:`absolute paths <dict absolute path>` to specify the :ref:`primary descriptor <dict primary descriptor file>`, :ref:`test parameter <dict parameter file>`, and readMePath files.
 
 Filled-out example of a single workflow without a name
 ------------------------------------------------------
-In this example, the workflow author is identified with an orcid. When an orcid is specified, there is no need to specify an author's name and email as that information will be pulled from the orcid. There are also three test parameter files given for the workflow.
+In this example, the workflow author is identified with an orcid. When an orcid is specified, there is no need to specify an author's name and email as that information will be pulled from the orcid. There are also three test parameter files given for the workflow. Since no readMePath is specified, Dockstore will show the top-level readme (if one is present), eg, ``./readme.md``
```

Comment:
1. It was like this, but since you're in here, if you want to change `orcid` to `ORCID`... See branding guidelines: https://info.orcid.org/brand-guidelines/
2. `./readme.md` -> `/readme.md` -- I only know that because I just reviewed line 13 :) (I know it's implicit here, but still).